docs/source/installation.rst

```diff
@@ -1,210 +1,159 @@
+After ``pip install ivadomed``, check if acceleration is available by running:
 
-                If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all),
-                please `install Python <https://wiki.python.org/moin/BeginnersGuide/Download/>`_ before continuing.
+::
 
-                Once you have a supported version of Python installed, run the following command:
+  $ python -c 'import torch; print(torch.cuda.is_available())'
+  True
 
-                .. tabs::
+If this reports 'False' or something else, you are not configured for GPU acceleration.
 
-                    .. group-tab:: Mac/Linux
+.. tabs::
 
-                        .. code::
+  .. group-tab:: Linux
 
-                            python3 -m venv ivadomed_env
+        To verify your GPU hardware is detected, use ``lspci`` like in:
 
-                        .. note::
+        ::
 
-                           If you use ``Debian`` or ``Ubuntu``, you may be prompted to install 
-                           the ``python3-venv`` module when creating the virtual environment.
-                           This is expected, so please follow the instructions provided by Python.
-                           For other operating systems, ``venv`` will be installed by default.
+            $ lspci -nn -d 10DE::0300
+            01:00.0 VGA compatible controller [0300]: NVIDIA Corporation GM200 [GeForce GTX TITAN X] [10de:17c2] (rev a1)
+            02:00.0 VGA compatible controller [0300]: NVIDIA Corporation GM200 [GeForce GTX TITAN X] [10de:17c2] (rev a1)
 
-                    .. group-tab:: Windows
+        To verify the NVIDIA driver, examine its version like this example:
 
-                        .. code::
+        ::
 
-                            python -m venv ivadomed_env
+            $ cat /sys/module/nvidia/version
+            510.73.05
 
-            2. Activate the new virtual environment (default named ``ivadomed_env``)
 
-                .. tabs::
+  .. group-tab:: Windows
 
-                    .. group-tab:: Mac/Linux
+        .. TODO: document verifying hardware/driver on Windows
 
-                        .. code::
+        The default Windows install is CPU-only, because it's still rare to be doing machine-learning
+        directly on Windows. But it's possible to use it by asking for a CUDA build:
 
-                            source ivadomed_env/bin/activate
+        .. NOTE: we must periodically update the URL here to use the most recent CUDA,
+                 or else pip will start prefering the PyPI build to the pytorch.org build
+        .. NOTE: here we recommend CUDA 10.2, because that matches what Linux currently gets from PyPI.
+                 If/when torch starts pushing CUDA 11 to PyPI, update this.
+        ::
 
-                    .. group-tab:: Windows
+            pip install ivadomed --force-reinstall --extra-index-url https://download.pytorch.org/whl/cu102
```

Comment:
In https://github.com/axondeepseg/axondeepseg/pull/642 I instead recommended people put this into pip.conf. I am leaning towards that being a better recommendation because the choice of which download.pytorch.org link to use really depends on people's hardware, and that doesn't change as rapidly as software, and if any other software they use uses pytorch...they're going to want to make the same call as for what build they want.